backend/src/core/types/s3.types.ts

```diff
@@ -0,0 +1 @@
+export type CSVParams = { [key: string]: string }
```

Comment:
Minor nitpick, should probably rename file to `parse-csv.types.ts`

Reply:
Thanks that's a good catch I forgot about this file.